src/frontends/paddle/src/op_utils.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+using namespace ov::frontend::paddle::op::default_opset;
+using namespace ov;
+using namespace ov::frontend;
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+Output<Node> get_tensor_list(const OutputVector& node) {
+    auto tensor_list = node;
+    for (size_t i = 0; i < node.size(); i++) {
+        if (node[i].get_shape().size() == 0) {
+            tensor_list[i] = std::make_shared<op::default_opset::Unsqueeze>(
+                node[i],
+                op::default_opset::Constant::create(element::i64, {1}, {0}));
+        }
+    }
+    Output<Node> res;
+    if (node.size() == 1) {
+        res = tensor_list[0];
+    } else {
+        res = std::make_shared<op::default_opset::Concat>(tensor_list, 0);
+    }
+    return res;
+}
+
+Output<Node> get_1d_tensor(const Output<Node>& node) {
```

Comment:
Do you meet any problem here without inserting unsqueeze node when node_dim == 0?